core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -259,8 +259,8 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-aws-ipi-ovn-custom-cidr:
     platform: aws
-  cucushift-installer-rehearse-aws-ipi-ovn-hypershift-cluster-bot:
-    platform: aws
+  cucushift-installer-rehearse-aws-ipi-ovn-hypershift:
+    platform: aws-qe
```

Comment:
Before this change can merge, you'll need to add `aws-qe` as a support platform in the clusterbot:
https://github.com/openshift/ci-chat-bot/blob/5305b6ceebe072fb2918b487b8747e109702e510/pkg/manager/prow.go#L56
or the workflow will almost immediately fail with an unsupported platform error.

Reply:
Thanks for the comment

Reply:
For future reference: 
This works because workflow-launch uses different validation logic. As [discussed](https://redhat-internal.slack.com/archives/CNHC2DK2M/p1722609379193539) in the channel, we'll keep the current SupportedPlatforms unchanged to avoid exposing the -qe platforms to the public.